content/member/li.md

```diff
@@ -0,0 +1,56 @@
++++
+bio = ""
+date = "2023-05-26"
+id = "li"
+interests = ["XAI","Machine Learning Applications"]
+name = "Li Liu"
+portrait = "/portraits/li.jpg"
+short_bio = "Li is a second-year Ph.D. student advised by Prof. Leilani in the Department of Computer Science and Engineering at UC Santa Cruz. His research interests are explaining the model intelligence and their applications in various contexts."
+short_name = "Li"
```

Comment:
We should figure out a better fix (eventually), but if you add `sort_position = 2` that will change the sorting issue.  The way it sorts is by date added, which we probably want to sort in reverse.